### ISSUE TITLE
Re-format dataset to not be a time series

### ISSUE THREAD

**Issue by apigatto** [BrettSimmons/DS301-Final-Project#1]
Average all observations for each country to make one observation.

**Comment by apigatto** [comment]
Make test set using avged observations for one year (2015), use rest of years as training set
